udata/linkchecker/backends.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+import logging
+import pkg_resources
+from datetime import datetime
+
+from flask import current_app
+
+log = logging.getLogger(__name__)
+
+
+ENTRYPOINT = 'udata.linkcheckers'
+
+
+class NoCheckLinchecker(object):
+    """Dummy linkchecker for resources that need no check"""
+
+    def check(self, _):
+        return {
+            'check:status': 204,
+            'check:available': True,
+            'check:date': datetime.now()
+        }
+
+
+def get(name):
+    '''Get a linkchecker given its name or fallback on default'''
+    all_lc = get_all()
```

Comment:
Not fond of shortened variables names but YMMV

Reply:
👍